src/Altinn.Platform/Altinn.Platform.Storage/UnitTest/TestingControllers/TextsControllerTests.cs

```diff
@@ -290,10 +291,13 @@ private HttpClient CreateTestHttpClient(ITextRepository textRepository)
             {
                 builder.ConfigureTestServices(services =>
                 {
+                    services.AddMockRepositories();
+
                     services.AddSingleton(textRepository);
                     services.AddSingleton(dataRepository.Object);
```

Comment:
I think the services set up in lines 297 - 299 will be the active ones rather than those defined in AddMockRepositories., so these lines should probably be deleted. 

Consider if it makes sense to send the mock used actively by the tests as input to the extension method or if it should be replaced in the test class as done here with textRepository. I can't quite decide what would be the better option.

Reply:
Yeah I don't think it makes sense here either, can remove it

Reply:
Removed `AddMockRepositories` call here now, but I'm not sure, the mocks above here don't actually have any behaviour setup..

If the mocked repositories aren't actually invoked in any way in these tests then `AddMockRepositories` might as well be used (no point in creating all the mock instances manually)

Reply:
Yeah, I would probably keep ```AddMockRepositories``` and remove the services that mock that don't hold any implementation. Saves us from the extra lines of code.